src/main.c

```diff
@@ -74,22 +98,28 @@ void left(uint8_t status, uint32_t held) {
     screenOn();
     if (!s.vsetLock){
       if(((held > 1000) && status & BUTTON_HELD)){
```

Comment:
This block could be redone as (pseudocoded):
`
} else if ((status & button_press) || ((held > 300) && ........) {
    if (held > 1000)
           leftUpdateInterval = 2;
    __down();
}
} else {
    screenOff();
    leftUpdateInterval = 100;
}
`

and it would look cleaner.